config/locales/en.account_components.yml

```diff
@@ -0,0 +1,16 @@
+# Labels for account components. These are often overridden/added to in the schools
+# to match their chart string fields (e.g. Fund, Org, Department, etc.)
+
+en:
+  nufs_account:
+    account_fields:
+      label:
+        account_number:
+          account_number: Account Number
+
+  facility_account:
+    # Components of the account
+    account_fields:
+      label:
+        account_number:
+          account_number: Account Number
```

Comment:
Should this be Payment Source Number to match where you moved it from, or is that an intentional change?

Reply:
The "Payment Source" string wasn't actually being used prior to the change that broke this, it was using "Account Number". It's not really that important because these components are only used in open since NU and DC both use their own chart string components.